airgun/views/dashboard.py

```diff
@@ -0,0 +1,162 @@
+from widgetastic.widget import (
+    Text,
+    View,
+    Widget,
+)
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    SatTable,
+)
+from airgun.widgets import (
+    ActionsDropdown,
+    PieChart,
+    Search,
+)
+
+
+class ItemValueList(Widget):
+    """List of name-value pairs. Each name element from every pair is clickable
+
+     Example html representation::
+
+    <ul>
+        <li>
+            <a class="dashboard-links"...>Hosts with no reports</a>
+            <h4>5</h4>
+        </li>
+     """
+    LABELS = ".//li/a[@class='dashboard-links']"
+    LABEL = ".//li/a[@class='dashboard-links'][normalize-space(.)='%s']"
+    VALUE = './/h4[preceding-sibling::a[contains(., "{}")]]'
+
+    def read(self):
+        """Return a dictionary where keys are widget criteria names and
+        values are number of hosts that correspond to these criteria
+        """
+        values = {}
+        for item in self.browser.elements(self.LABELS):
+            name = self.browser.text(item)
+            value = self.browser.text(self.VALUE.format(name))
+            values[name] = value
+        return values
+
+    def fill(self, value):
+        """Click on specific criteria from the widget list"""
+        self.browser.click(
+            self.browser.element(self.LABEL % value))
+
+
+class TotalCount(Widget):
+    """Return total hosts count from Host Configuration Status type of
+    widgets
+    """
+    total_count = Text(".//h4[@class='total']")
+
+    def read(self):
+        """Return hosts count from widget. Usually it is a string like
+        'Total Hosts: 5'
+        """
+        _, count = self.total_count.read().split(': ')
```

Comment:
we can omit the space 
```
 _, count = self.total_count.read().split(':')
```
but in this concrete usage prefere partition usage
```
_, _, count = self.total_count.read().partition(':')
```
"not a blocker"